Makefile

```diff
@@ -298,32 +302,32 @@ clean-test-results:
 
 build-tests:
 	@printf $(COLOR) "Build tests..."
-	@go test -exec="true" -count=0 $(TEST_DIRS)
+	@go test $(BUILD_TAG_FLAG) -exec="true" -count=0 $(TEST_DIRS)
```

Comment:
probably should be TEST_TAG_FLAG to match the flags used when actually running the tests (one purpose of this target is to be a pre-build to get stuff in cache)

Reply:
Thanks, I've fixed this